src/otg/gwas_catalog_ingestion.py

```diff
@@ -30,13 +33,15 @@ class GWASCatalogIngestionStep:
         ld_populations (list): List of populations to include.
         catalog_studies_out (str): Output GWAS catalog studies path.
         catalog_associations_out (str): Output GWAS catalog associations path.
+        gwas_catalog_study_curation_file (str | None): file of the curation table. Optional.
```

Comment:
I don't see that much confusion there. GWAS Catalog indeed curate studies, but all the studies we ingest from GWAS Catalog are curated, so there's not ambiguity because we just call them `catalog_studies`. Study curation mentioned only one place.